phpmyfaq/sitemap.xml.php

```diff
@@ -61,7 +61,7 @@
 // {{{ Functions
 function buildSiteMapNode($location, $lastModified = null)
 {
-    if (!isset($lastModified)) {
+    if ( empty($lastModified) ) {
```

Comment:
Could you please remove the blank space before empty()

Reply:
Yes, of course ;-)